codec_specs.md

```diff
@@ -623,7 +645,9 @@ Codec ID: A_WAVPACK4
 
 Codec Name: [WavPack](http://www.wavpack.com/) lossless audio compressor
 
-Description: The Wavpack packets consist of a stripped header followed by the frame data. For multi-track (> 2 tracks) a frame consists of many packets. For hybrid files (lossy part + correction part), the correction part is stored in an additional block (level 1). For more details, check the [WavPack muxing description](wavpack.html).
+Description: The Wavpack packets consist of a stripped header followed by the frame data. For multi-track (> 2 tracks) a frame consists of many packets. For more details, check the [WavPack muxing description](wavpack.html).
+
+Codec BlockAdditions: For hybrid A_WAVPACK4 encodings (that include a lossy encoding with a supplemental correction to produce a lossless encoding), the correction part is stored in BlockAdditional.
```

Comment:
backtick A_WAVPACK4

Reply:
Maybe we need to mention it's the `BlockAdditional` data marked with `BlockAddID`=1 ? (same for VP8 and VP9).